src/server/index.ts

```diff
@@ -21,13 +21,13 @@ function analyzer(opts: AnalyzerPluginOptions = defaultAnalyzerOptions): Plugin
   const { reportTitle = name } = opts
   const analyzerModule = createAnalyzerModule(opts?.gzipOptions)
   let defaultWd = process.cwd()
-  
+
   const plugin = <Plugin>{
     name,
     apply: 'build',
     enforce: 'post',
     configResolved(config) {
-      defaultWd = config.root
+      defaultWd = config.build.outDir ?? config.root
```

Comment:
Can you explain why `config.build.outDir`? In my opinion at present we can see the generated results in the current directory

Reply:
I think an outdir a best place instead root because: 
- Autoremoving for each building process 
- If the application has multiple build parallel runs we avoid a file result overwriting like a rat race. ( For example, I have an app that generates several bundles with different input vite options and I want to know the bundle analysis result for all of them )
- We can rewrite a client file with the same name that it contains in root dir
In my opinion, all generated files should be contained in an isolated place that does not exist in normal cases on the client side by default ( for example dist folder ).

Reply:
It seems you are right :)